### PR TITLE
Ensured Nav appears on edit institution scree #1893

### PR DESCRIPTION
**Pull request by DaveBlueFrontier**
All tests passed

### CODE REVIEW

File: app/controllers/institutions_controller.rb [hunk 1]
```diff
@@ -30,6 +32,7 @@ def new
   end
 
   def create
+    @usenav = true
```
Comment (tripledub) [inline app/controllers/institutions_controller.rb]: Don't need this here are `:create` action won't render a view, it either redirects or re-renders `:new` on error

Comment (DaveBlueFrontier) [inline app/controllers/institutions_controller.rb]: Just me being over-zealous.

Comment (tripledub) [inline app/controllers/institutions_controller.rb]: Can you remove it and I'll merge it straight away

Comment (tripledub) [inline app/controllers/institutions_controller.rb]: @DaveBlueFrontier bump!!

**Pull request merged**